index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="stylesheet" href="css/style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="mob-navbar">
+        <a href="#">Welcome</a>
+        <img src="image/Union.png" alt="option">
+      </div>
+    </nav>
```

Comment:
- [ optional] only the hamburger icon should be in the `nav tag.` the logo should be on its own. please look into that